pkg/upgrade/upgrader/helper.go

```diff
@@ -62,3 +64,25 @@ func GetPatchData(oldObj, newObj interface{}) ([]byte, error) {
 	}
 	return patchBytes, nil
 }
+
+func isOperatorUpgraded(componentName string, namespace string,
+	toVersion string, kubeClient kubernetes.Interface) error {
+	operatorPods, err := kubeClient.CoreV1().
+		Pods(namespace).
+		List(metav1.ListOptions{
+			LabelSelector: "openebs.io/component-name=" + componentName,
+		})
+	if err != nil {
+		return err
+	}
+	if len(operatorPods.Items) == 0 {
+		return fmt.Errorf("operator pod missing for %s", componentName)
+	}
+	for _, pod := range operatorPods.Items {
+		if pod.Labels["openebs.io/version"] != toVersion {
```

Comment:
nit: We can move `openebs.io/version` to some global constant.

Reply:
Sure. Will take this in separate PR.